CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+# Contributing
+
+Thank you for considering contributing to Pactus blockchain!
+Please read these guidelines before submitting a pull request or opening an issue.
+
+## Code guidelines
+
+We strive to maintain clean, readable, and maintainable code in Pactus blockchain.
+Please follow these guidelines when contributing code to the project:
+
+- Code should follow the [Effective Go](https://golang.org/doc/effective_go.html) guidelines.
+- Documentation should follow the [Go Doc Comments](https://go.dev/doc/comment) format.
+- Use clear and descriptive variable, function, and method names. Avoid using abbreviations or acronyms unless they are well-known and widely understood.
+- Write code that is modular and reusable, and follows the [DRY (Don't Repeat Yourself)](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) principle.
```

Comment:
remove this line

Reply:
Link to this document: https://gist.github.com/wojteklu/73c6914cc446146b8b533c0988cf8d29